Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -120,7 +120,7 @@ class VLCMediaCategoryViewController: UICollectionViewController, UICollectionVi
     func setupEditToolbar() {
         editToolbar.translatesAutoresizingMaskIntoConstraints = false
         view.addSubview(editToolbar)
-        editToolbarConstraint = editToolbar.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 50)
+        editToolbarConstraint = editToolbar.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 60)
```

Comment:
according to figma it's 50. If you want to account for the shadow as well it's 50 + 3 I believe ? Rest is fine and can be merged. I would just double check that

Reply:
No, with 53 you can still see the shadow. So to have a bit of margin, I decided to make make it 60.